### ISSUE TITLE
Issue Importing into blender, Rigfy option + Cycles 

### ISSUE THREAD

**Issue by FractalMoon** [FlailingFog/KK-Blender-Porter-Pack#353]
I have no particular issue importing from KK to blender. KK or Rigfy armatures works, fine the problem is very specific when Importing a character with Rigfy armature with the cycles shader option, making the clothing orientation and scale correctly but not for the body nor the bones.

Importing with the KK armature in cycles works fine but I wonder what has the shader to do with the rigfy armature that turns out that messed up. It happens to me in every character, This is something speciffic to me or is a known issue? 

There is any way to fix this?

![R1jjnZ6](https://user-images.githubusercontent.com/42303712/214268546-1070b9da-b648-46e7-919a-bd9d32a31bad.png)

![LQ3Svkz](https://user-images.githubusercontent.com/42303712/214269076-589c438e-a249-4839-aaf1-57d22df8b1ff.png)

Thank you for reading.

**Comment by ilcane87** [comment]
Hi @FractalMoon, I checked and you are right, there is an issue with importing with the settings you mentioned, and I can't really tell what the cause is, but fortunately there is an easy way you can achieve the same result without errors: just import the model with the Cycles option like before, but this time choose the standard _KKBP Armature_ option instead of _Rigify Armature_.

Then, after the character is done loading and before making any changes, select the armature in object mode and click the _Convert for Rigify_ button in the _Extras_ section of the addon:
![2023-01-24 14_17_47-Blender](https://user-images.githubusercontent.com/34897042/214318450-70b23e37-1d0e-4df4-9809-3ff4a988ae1d.png)
This will covnert the standard armature to the Rigify armature, just like when you import by selecting Rigify directly, but this time it won't have the weird behavior you got before.

**Comment by FractalMoon** [comment]
Hey @ilcane87 thank you for your time the fix did the trick but with workarround. the solution didn't work for me instantly but it seems that one loading the rigfy for the first time on KK Armature did not work but by just Ctl+Z and runing the rigfy converter over it agin did the job. I converted like 10 character and I had to load the rigfy converter twice everytime.

I add this just becaus anyone is eventually come here with the same isue so that's how I have fixed it.

Thank you again ilcane87 for telling me about the rigfy converter on the extras tab I haven't noticed it before.
